### PR TITLE
fixed P4Runtime specs links

### PR DESCRIPTION
**Pull request by EmanueleGallone**
Hi,

just a quick fix to the links showing P4Runtime specs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ Git tags are used to mark minor and patch release versions.
...
-The latest version of the P4Runtime v1 specification is available:
-* [here](https://p4.org/p4runtime/spec/main/P4Runtime-Spec.html) in **HTML**
+The latest version of the P4Runtime v1.3 specification is available:
+* [here](https://p4lang.github.io/p4runtime/spec/v1.3.0/P4Runtime-Spec.html) in **HTML**
   format
-* [here](https://p4.org/p4runtime/spec/main/P4Runtime-Spec.pdf) in **PDF**
+* [here](https://p4lang.github.io/p4runtime/spec/v1.3.0/P4Runtime-Spec.pdf) in **PDF**
   format
 
 It is updated every time a new commit is pushed to the main branch.
```
Comment (antoninbas) [inline README.md:29]: This is supposed to be the version generated from the main branch:

> It is updated every time a new commit is pushed to the main branch.

**Review by antoninbas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EmanueleGallone) [reply README.md:29]: Hi, this was intended as a temporary fix, at least until you solved the issues with the p4.org links. Should I just close this pull request?

Comment (antoninbas) [reply README.md:29]: I think the PR is fine, but the links should be:
* https://p4lang.github.io/p4runtime/spec/main/P4Runtime-Spec.html
* https://p4lang.github.io/p4runtime/spec/main/P4Runtime-Spec.pdf

File: README.md [hunk 2]
```diff
@@ -20,10 +20,10 @@ Git tags are used to mark minor and patch release versions.
 
 ## Reading the latest version of the documentation
 
-The latest version of the P4Runtime v1 specification is available:
-* [here](https://p4.org/p4runtime/spec/main/P4Runtime-Spec.html) in **HTML**
+The latest version of the P4Runtime v1.3 specification is available:
```
Comment (antoninbas) [inline README.md:23]:
```suggestion
The latest version of the P4Runtime v1 specification is available:
```

**Review by antoninbas — APPROVED** [review]

**Pull request merged**